integration/addon_test.go

```diff
@@ -407,6 +407,76 @@ func (s *integrationTestSuite) TestAddonConditions() {
 		s.Require().Equal(addonsv1alpha1.AddonReasonMissingCSV, availableCond.Reason)
 	})
 
+	s.Run("sets_installed=true_with_install_ack_required", func() {
+		// remove addon before starting the test.
+		s.addonCleanup(addon, ctx)
+
+		addon := addonWithInstallAck()
+
+		err := integration.Client.Create(ctx, addon)
+		s.Require().NoError(err)
+
+		// assert that the installed condition is present and is set to false, when the
+		// addon is being installed.
+		err = integration.WaitForObject(
```

Comment:
Just checking the installed condition is not enough here, what we want to test is that the installed condition stays false even after the addon's CSV is ready.

Reply:
Thanks, I've modified the test to now watch for the addon's CSV prior.